lua/gitblame/git.lua

```diff
@@ -15,15 +15,20 @@ function M.check_is_ignored(callback)
     })
 end
 
+local function get_http_domain(url)
```

Comment:
Nitpick just for clarity:

```suggestion
---@return string?
local function get_http_domain(url)
```